content/compose/compose-file/05-services.md

```diff
@@ -20,7 +20,58 @@ available resources. Deploy support is an optional aspect of the Compose Specifi
 described in detail in the [Compose Deploy Specification](deploy.md) documentation.
 If not implemented the `deploy` section is ignored and the Compose file is still considered valid.
 
-## annotations
+## Examples
+
+### Simple example
+
+```yaml
+services:
+  web:
+    image: nginx:latest
+    ports:
+      - "8080:80"
+
+  db:
+    image: postgres:13
+    environment:
+      POSTGRES_USER: example
+      POSTGRES_PASSWORD: example
```

Comment:
Not sure we should show this kind of setup, users will reproduce it and it's definitely not a good practice, we should use a secret for this or at least setup the password value via an env variable. 
for example
```suggestion
      POSTGRES_PASSWORD: ${POSTGRES_USER_PASSWORD}
```
Or just remove the `POSTGRES_PASSWORD` definition in this example